### PR TITLE
10 account move import atos

### PR DESCRIPTION
**Pull request by sebastienbeau**

### CODE REVIEW

File: account_move_import_atos/models/account_journal.py [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+
+# from odoo.tools.translate import _
+from odoo import models, fields
+
+
+class AccountJournal(models.Model):
+    _name = 'account.journal'
+    _inherit = ['account.journal', 'mail.thread']
```
Comment (sebastienbeau) [inline account_move_import_atos/models/account_journal.py]: pas besoin du inherit sur mail.thread

File: account_move_import_atos/models/account_journal.py [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+
+    import_type = fields.Selection(
+        selection_add=[
+            ('atos_csvparser', 'Parser for Atos/Sips (mercanet) xls file')])
+
+    # def _add_special_line(self, cursor, uid, statement_id, parser, result_row_list, profile, context=None):
```
Comment (sebastienbeau) [inline account_move_import_atos/models/account_journal.py]: a supprimer

File: account_move_import_atos/parser/atos_parser.py [hunk 3]
```diff
@@ -0,0 +1,133 @@
...
+            'transaction_ref': line["TRANSACTION_ID"],
+        }
+        return res
+
+    def _post(self, *args, **kwargs):
+        """
+        Compute the total transfer amount
+        """
+        res = super(AtosFileParser, self)._post(*args, **kwargs)
+        self.transfer_amount = 0.0
```
Comment (sebastienbeau) [inline account_move_import_atos/parser/atos_parser.py]: transfer and refund amount are not necessary anymore

File: account_move_import_atos/parser/atos_parser.py [hunk 4]
```diff
@@ -0,0 +1,133 @@
...
+                raise osv.except_osv(
+                    _("User Error"),
+                    _("The bank statement imported have invalide line,"
+                      " indeed the operation type %s is not supported")
+                    % row['OPERATION_NAME'])
+        self.result_row_list = rows
+        self.statement_date = self.result_row_list[0]["OPERATION_DATE"]
+        return res
+
+    # def get_refund_amount(self):
```
Comment (sebastienbeau) [inline account_move_import_atos/parser/atos_parser.py]: remove useless comment

File: account_move_import_atos/parser/atos_parser.py [hunk 5]
```diff
@@ -0,0 +1,133 @@
...
+                    'credit':value,
+                    'debit':value
+                }
+        """
+        amount = line['OPERATION_AMOUNT']
+        credit = bool(line['OPERATION_NAME'] == 'CREDIT_CAPTURE')
+        res = {
+            'name': line["OPERATION_NAME"] + '_' + line["TRANSACTION_ID"],
+            'date_maturity': line["OPERATION_DATE"],
+            'credit': amount if credit else 0.0,
```
Comment (sebastienbeau) [inline account_move_import_atos/parser/atos_parser.py]: A inverser

File: account_move_import_atos/parser/atos_parser.py [hunk 6]
```diff
@@ -0,0 +1,133 @@
...
+        """
+        res = super(AtosFileParser, self)._post(*args, **kwargs)
+        self.transfer_amount = 0.0
+        self.refund_amount = 0.0
+        rows = []
+        for row in self.result_row_list:
+            if row['OPERATION_NAME'] in ('CREDIT'):
+                continue
+            rows.append(row)
+            if row['OPERATION_NAME'] == 'CREDIT_CAPTURE':
```
Comment (sebastienbeau) [inline account_move_import_atos/parser/atos_parser.py]: not needed anymore as we have a debit and a credit

**Review by PierrickBrun — APPROVED** [review]

**Pull request merged**